### PR TITLE
Fixes to docs/concepts/vault_plugins.md

### PR DESCRIPTION
**Pull request by revit13**
This PR contains fixes to docs/concepts/vault_plugins.md.

Thanks to @elsalant for reporting.

### CODE REVIEW

File: site/docs/concepts/vault_plugins.md [hunk 1]
```diff
@@ -7,14 +7,20 @@ This project uses secrets plugins to retrieve dataset credentials by the running
 
 Additional secret plugins can be developed to retrieve credentials additional location. This [tutorial](https://learn.hashicorp.com/tutorials/vault/plugin-backends?in=vault/app-integration) can serve as a good starting point to learn about Vault plugin development.
 
-The following steps are for configuring a secret plug-in for Fybrik:
+## Before you begin
 
-1. Enable the plugin during Vault server initialization in a specific path. 
-<br/>An example of that can be found in helm chart [values.yaml](https://github.com/fybrik/fybrik/blob/master/third_party/vault/vault-single-cluster/values.yaml) file in the project where [Vault-plugin-secrets-kubernetes-reader](https://github.com/fybrik/vault-plugin-secrets-kubernetes-reader) plugin is enabled in `kubernetes-secrets` path:
+Ensure that you have the [Vault v1.9.x](https://www.vaultproject.io/docs/commands) to execute Vault CLI commands.
```
Comment (elsalant) [inline site/docs/concepts/vault_plugins.md:12]: I don't think this is pointing to the right location - shouldn't the reference be:
https://www.vaultproject.io/downloads

File: site/docs/concepts/vault_plugins.md [hunk 2]
```diff
@@ -23,7 +29,7 @@ vault policy write "allow-all-dataset-creds" - <<EOF
       }
       EOF
 ```
-3. Have the `CatalogDatasetInfo` structure from the [data catalog response](../../reference/connectors#data_catalog_responseproto) contain the Vault secret path which should be used to retrieve the credentials for a given asset. When Vault plugin is used to retrieve the credentials the parameters to the plugin should follow the plugin usage instructions. This path will later be passed on to the [modules](./modules.md).
+4. Have the `CatalogDatasetInfo` structure from the [data catalog response](../../reference/connectors#data_catalog_responseproto) contain the Vault secret path which should be used to retrieve the credentials for a given asset. When Vault plugin is used to retrieve the credentials the parameters to the plugin should follow the plugin usage instructions. This path will later be passed on to the [modules](./modules.md).
```
Comment (elsalant) [inline site/docs/concepts/vault_plugins.md:32]: "When Vault plugin" ->"When the Vault plugin"

File: site/docs/concepts/vault_plugins.md (hunk 1 above)
Comment (revit13) [reply site/docs/concepts/vault_plugins.md]: Fixed. Thanks

File: site/docs/concepts/vault_plugins.md (hunk 2 above)
Comment (revit13) [reply site/docs/concepts/vault_plugins.md]: Fixed. Thanks

**Review by elsalant — APPROVED** [review]

**Pull request merged**